taxonomies/ingredients.txt

```diff
@@ -33137,7 +33154,7 @@ cy:Llugaeron
 da:Tranebær
 de:Moosbeeren, Cranberry, Cranberrys
 eo:Oksikoko
-es:arándano rojo, arándano
+es:arándano rojo, arándano,arandano,arandanos
```

Comment:
arandano,arandanos will not be needed when deaccenting is fixed (soon :) )

Reply:
ok removed - fingers crossed....